src/helpers.js

```diff
@@ -100,3 +108,30 @@ export function runQuery (instance, query, needResponse) {
     return response
   })
 }
+
+export function findLastObject (instance, table) {
+  return new Promise((resolve, reject) => {
+    let queryInfo = `PRAGMA table_info("${table}")`
+    let queryId = `SELECT seq FROM sqlite_sequence WHERE name="${table}"`
```

Comment:
Yeah, but you know what? (I guess you're referring that I'm not using option 2 (`select last_insert_rowid()`)) I realized that's not possible to know what is the column name to use with `findOne()`. So I think we should give up if option 1 fails. Or what would be the search parameters?